dev/breeze/src/airflow_breeze/prepare_providers/provider_documentation.py

```diff
@@ -1066,6 +1081,7 @@ def get_provider_documentation_jinja_context(
     )
     jinja_context["WITH_BREAKING_CHANGES"] = with_breaking_changes
     jinja_context["MAYBE_WITH_NEW_FEATURES"] = maybe_with_new_features
+    jinja_context["MAYBE_WITH_CHANGE_MIN_AIRFLOW_VERSION"] = maybe_with_change_min_airflow_version
```

Comment:
Can you add a print statement here to print the jinja_context once? It will be clear.

Reply:
It evaluates to False this is the issue :)
I want it to be true when I select at least one PR with `v`

Reply:
Yeah understood.